otx/algorithms/segmentation/configs/ocr_lite_hrnet_18/semisl/__init__.py

```diff
@@ -0,0 +1,15 @@
+"""Initialization of OCR-Lite-HRnet-18 model for Semi-SL Segmentation Task."""
```

Comment:
I think we don't need semi-sl for `OCR-Lite-HRNet-18` architecture, this backbone is deprecated. We only need to leave template.yaml to be scanned well the template id when the user upgrade from Geti 1.0 to Geti 1.1